code/modules/surgery/organs/robolimbs.dm

```diff
@@ -3,142 +3,223 @@ GLOBAL_LIST_EMPTY(chargen_robolimbs)
+	desc = "This limb has a polished metallic casing and a holographic face emitter."
+	loadout_name = "Rook"
+	icon = 'icons/mob/human_races/cyberlimbs/bishop/bishop_rook.dmi'
+	has_subtypes = brand
+
+/datum/robolimb/rook/monitor
+	company = "Bishop Cybernetics Rook Monitor"
 	icon = 'icons/mob/human_races/cyberlimbs/bishop/bishop_monitor.dmi'
 	parts = list("head")
-	is_monitor = 1
-	selectable = 0
-	has_subtypes = null
+	is_monitor = TRUE
+	selectable = FALSE
+	has_subtypes = model
 
+/* Hesphiastos */
+//Main
 /datum/robolimb/hesphiastos
 	company = "Hesphiastos Industries"
 	desc = "This limb has a militaristic black and green casing with gold stripes."
+	loadout_name = "Hesphiastos"
 	icon = 'icons/mob/human_races/cyberlimbs/hesphiastos/hesphiastos_main.dmi'
-	has_subtypes = 1
-
-/datum/robolimb/hesphiastos/alt1
-	company = "Hesphiastos Industries alt."
-	icon = 'icons/mob/human_races/cyberlimbs/hesphiastos/hesphiastos_alt1.dmi'
-	parts = list("head")
-	is_monitor = 1
-	selectable = 0
-	has_subtypes = null
+	has_subtypes = brand
 
 /datum/robolimb/hesphiastos/monitor
-	company = "Hesphiastos Industries mtr."
+	company = "Hesphiastos Industries Monitor"
 	icon = 'icons/mob/human_races/cyberlimbs/hesphiastos/hesphiastos_monitor.dmi'
 	parts = list("head")
-	is_monitor = 1
-	selectable = 0
-	has_subtypes = null
+	is_monitor = TRUE
+	selectable = FALSE
+	has_subtypes = model
+
+//Titan
+/datum/robolimb/titan
+	company = "Hephiastos Industries Titan"
+	desc = "This limb has a casing of an olive drab finish, providing a reinforced housing look."
+	loadout_name = "Titan"
+	icon = 'icons/mob/human_races/cyberlimbs/hesphiastos/hesphiastos_titan.dmi'
+	has_subtypes = brand
+
+/datum/robolimb/titan/monitor
+	company = "Hesphiastos Industries Titan Monitor"
+	icon = 'icons/mob/human_races/cyberlimbs/hesphiastos/hesphiastos_alt1.dmi'
+	parts = list("head")
+	is_monitor = TRUE
+	selectable = FALSE
+	has_subtypes = model
 
+/* Morpheus */
+//Main
 /datum/robolimb/morpheus
 	company = "Morpheus Cyberkinetics"
 	desc = "This limb is simple and functional; no effort has been made to make it look human."
+	loadout_name = "Morpheus"
 	icon = 'icons/mob/human_races/cyberlimbs/morpheus/morpheus_main.dmi'
-	unavailable_at_chargen = 1
-	is_monitor = 1
-	has_subtypes = 1
+	has_subtypes = brand
 
-/datum/robolimb/morpheus/alt1
-	company = "Morpheus Cyberkinetics alt."
+/datum/robolimb/morpheus/monitor
+	company = "Morpheus Cyberkinetics Monitor"
 	icon = 'icons/mob/human_races/cyberlimbs/morpheus/morpheus_alt1.dmi'
 	parts = list("head")
-	unavailable_at_chargen = null
-	is_monitor = null
-	selectable = 0
-	has_subtypes = 2 //Edge case. We want to be able to pick this one, and if we had it left as null for has_subtypes we'd be assuming it'll be chosen as a child model,
-					 //and since the parent is unavailable at chargen, we wouldn't be able to see it in the list anyway. Now, we'll be able to select the Morpheus Ckt. Alt. head as a solo-model.
```

Comment:
This is what I need to pay the most attention to. @Kiyahitayika 
I changed this from 2 (childless) to null (model).

I honestly did not understand the comment that was left and I should have read it more thoroughly,  but I was too invested in my concept of cleaning. I think the 2 here translates over to preferences.dm and allows this Monitor to be the Default IPC monitor in character setup. Time for a test with it as null :P